### PR TITLE
DOC: Small one. The input to this one needs to be 2D.

### PR DESCRIPTION
**Pull request by arokem**

### CODE REVIEW

**Comment by emmanuelle** [comment]
Thanks @arokem 

While you're at it, would you mind
- testing for the number of dimensions of the image and raising a ValueError for ``ndim > 2``?
- and writing a test in the corresponding test module to be sure that the function fails gracefully with non 2-D images?

If you don't have the time, it's no big deal. We can also merge and work later on further improvements.

**Comment by arokem** [comment]
Yeah. Happy to do it. Give me a few.

**Comment by emmanuelle** [comment]
great, thanks a lot!

**Comment by arokem** [comment]
OK - how is this?

**Comment by emmanuelle** [comment]
Looks awesome! Just waiting for Travis to finish before merging.

File: skimage/morphology/tests/test_convex_hull.py [hunk 1]
```diff
@@ -31,20 +31,24 @@ def test_basic():
 
     assert_array_equal(convex_hull_image(image), expected)
 
+    # Test that an error is raised on passing a 3D image:
+    image3d = np.empty((5, 5, 5))
+    assert_raises(ValueError, convex_hull_object, image3d)
```
Comment (emmanuelle) [inline skimage/morphology/tests/test_convex_hull.py]: is it the same test as L148 below? Do you mean ``convex_hull_image`` here?

Comment (arokem) [inline skimage/morphology/tests/test_convex_hull.py]: Whoops. Good catch. Serves me for copy-pasting!

**Comment by emmanuelle** [comment]
Merging, thanks!

**Pull request merged**